src/universe/analytics.ts

```diff
@@ -99,6 +111,27 @@ export function analytics (this: Universe): UniverseAnalytics {
     },
     subscriberMetrics: async (options?: UniverseAnalyticsOptions): Promise<SubscriberMetrics> => {
       return await makeAnalyticsRequest<SubscriberMetrics, undefined>('/universe/subscriptions')
+    },
+    revenueMetrics: async (options?: UniverseRevenueMetricsOptions): Promise<RevenueMetrics> => {
+      if (!options?.start || !options?.end) { throw new RevenueMetricsStartEndMissingError() }
+      if (!options?.version) { throw new RevenueMetricsVersionError() }
```

Comment:
todo: Instead of runtime errors, I prefer just to set the options not to be optional. That way, we cannot write code that does not specify the required fields.